admin/index.php

```diff
@@ -32,5 +32,11 @@
 if ($foundAction) {
     exit(0);
 } else {
-    require(preg_replace('/[^a-zA-Z0-9_-]/', '', $cmd) . '.php');
+    $filename = preg_replace('/[^a-zA-Z0-9_-]/', '', $cmd) . '.php';
+    if (file_exists($filename)) { 
+       require(preg_replace('/[^a-zA-Z0-9_-]/', '', $cmd) . '.php');
```

Comment:
can this second `preg_replace` be replaced with `$filename`?

Also, `require` doesn't require parentheses. We're progressively removing parentheses for require/include statements and their `_once` equivs.